### ISSUE TITLE
Monospace font for the edit section of Manage Tabs

### ISSUE THREAD

**Issue by Mollerz** [thewca/worldcubeassociation.org#1841]
It's a pain making large tables and not having it monospaced! There is a toggle preview mode so that you can see how it is for when it isn't monospaced, or to see how the markdown has been applied. I can't see anyone having a problem with it being monospaced, but definitely having it as an option will be very useful.

**Comment by jfly** [comment]
I agree that a monospaced font would definitely be nice here, but I don't know if trying to change the textbox font would screw with what [simplemde](https://github.com/sparksuite/simplemde-markdown-editor) does for fancy styling as you edit. Someone will just have to give it a shot and see =)

**Comment by dunkOnIT** [comment]
Closing due to inactivity
